src/main/java/de/azapps/kafkabackup/common/record/RecordJSONSerde.java

```diff
@@ -0,0 +1,36 @@
+package de.azapps.kafkabackup.common.record;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import de.azapps.kafkabackup.common.AlreadyBytesConverter;
+import org.apache.kafka.common.record.TimestampType;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaAndValue;
+import org.apache.kafka.connect.header.ConnectHeaders;
+import org.apache.kafka.connect.header.Header;
+import org.apache.kafka.connect.header.Headers;
+import org.apache.kafka.connect.storage.Converter;
+
+import java.util.Map;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+
+public class RecordJSONSerde {
+    public static Record read(InputStream inputStream) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        SimpleModule module = new SimpleModule();
+        module.addDeserializer(Record.class, new RecordJSONDeserializer());
+        mapper.registerModule(module);
```

Comment:
That is indeed the question. I'm not sure how it will affect thread-safety.
Note that the methods are static, but I assume that we would in this case actually instantiate the class and move this stuff to constructor right?
And then we can create a RecordJSONSerde per thread and re-use it. Sounds good?

Reply:
ObjectMapper can be a singleton and is thread-safe - so maybe let's move it to the static field?

Reply:
In the end I opted for not making it static, after reading this: https://stackoverflow.com/a/36162525
What do you say about the current solution?